src/inter-aut.cc

```diff
@@ -601,7 +599,11 @@ bool Mata::IntermediateAut::is_graph_conjunction_of_negations(const Mata::Formul
             return false;
     }
 
-    return true;
+    if (act_graph->node.is_operator() && act_graph->node.is_neg()) {
```

Comment:
I Believe this could be shortened to `return act_graph->node.is_operator() && act_graph->node.is_neg()`

Reply:
Yeah that's true, fixed